app/controllers/application_controller.rb

```diff
@@ -37,7 +37,9 @@ def set_active_banner
     return nil unless current_organization
 
     @active_banner = current_organization.banners.active.first
+
     @active_banner = nil if session[:dismissed_banner] == @active_banner&.id
+    @active_banner = nil if @active_banner&.expires_at && Time.now.in_time_zone(cookies[:browser_time_zone]) > @active_banner&.expires_at
```

Comment:
Let's extract this to a single method and then test against it.

Reply:
extracted it to `Banner#expired` and added a test. Also switched to `Time.current` per https://thoughtbot.com/blog/its-about-time-zones.

I also realized that we don't need the timezone conversion here because `Time.current`/`Time.now` are computed in the backend and `>` is timezone aware, so I removed that as well.